dotcom-rendering/src/web/components/CardHeadline.tsx

```diff
@@ -216,6 +216,9 @@ const WithLink = ({
 	return <>{children}</>;
 };
 
+/** Matches headlines starting with short words of 1 to 3 letters followed by a space */
+const isFirstWordShort = /^(\w{1,3})\b/;
```

Comment:
This'll do it, @bryophyta:

```suggestion
const isFirstWordShort = /^(\w{1,3}) /;
```